packages/lore-generate-new/templates/server.js

```diff
@@ -12,6 +12,18 @@
 var webpack = require('webpack');
 var WebpackDevServer = require('webpack-dev-server');
 var config = require('./webpack.config');
+var url = require('url');
+var _ = require('lodash');
+
+function getDevServerPort() {
```

Comment:
This function is here to prevent having to set the default port twice, and also because there's no easy way to extract the port the development server will be using. And `webpack.config.js` has to be an object, so it can't be passed variables (unless the config was moved to it's own file which I don't want to do because it'd be the only variable in the file....unless we created a `config/webpack.js` file maybe).